lib/Terminal/Print.pm6

```diff
@@ -99,9 +94,39 @@ class Terminal::Print {
         }
     }
 
-    method root-widget() {
+    method setup(:$reset = False) {
```

Comment:
What about resize *without* deleting contents?

Reply:
Not sure about preserving the content. It feels more correct for the content to be re-generated as the layout must change. This is a reason why I didn't move `SIGWINCH` processing in `Terminal::Print` and left it up to an app developer. As soon as it is known that terminal size changed he/she can take measures either to restore the old content or repaint from scratch.